activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -97,12 +97,7 @@ def increment(name, amount = 1, options = nil) # :nodoc:
           value
         end
 
-        def decrement(name, amount = 1, options = nil) # :nodoc:
-          return super unless local_cache
-          value = bypass_local_cache { super }
-          write_cache_value(name, value, options)
-          value
-        end
+        alias_method :decrement, :increment
```

Comment:
(The implementations do look the same, but they're `super`ing to different methods.)